app/scripts/modules/core/src/pipeline/config/health/stagePlatformHealthOverrideWrapper.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { IComponentOptions, module } from 'angular';
+
+export class StagePlatformHealthOverrideWrapperDirective implements IComponentOptions {
```

Comment:
I don't think this wrapper component is actually necessary.  The wrapper is necessary to adapt directives using `@` binding for example, but this wrapper component bindings are all `<`.  

See how we did `angular2react` for the `runningTasksTag` directive.

https://github.com/spinnaker/deck/blob/8bb02f358fffd096f72aa9fe895ba9703f489958/app/scripts/modules/core/src/reactShims/ngReact.ts#L65

https://github.com/spinnaker/deck/blob/532ab7784ca93569308c8f2ab80a18d313b910f9/app/scripts/modules/core/src/serverGroup/pod/runningTasksTag.directive.js#L19